doc/onboarding.md

```diff
@@ -64,31 +64,29 @@ onboarding session.
     * will also come more naturally over time
 
 
-  * reviewing:
-    * primary goal is for the codebase to improve
-    * secondary (but not far off) is for the person submitting code to succeed
-      * helps grow the community
-      * and draws new people into the project
```

Comment:
Did you feel these two points were redundant/any other specific reason to drop them?